modules/openapi-generator/src/main/resources/typescript-fetch/modelEnum.mustache

```diff
@@ -11,3 +11,6 @@ export function {{classname}}FromJSONTyped(json: any, ignoreDiscriminator: boole
 export function {{classname}}ToJSON(value?: {{classname}} | null): any {
     return value as any;
 }
+export function {{classname}}ToJSONTyped(value: any, ignoreDiscriminator: boolean): {{classname}} {
```

Comment:
Please use the same parameter order as in modules/openapi-generator/src/main/resources/typescript-fetch/modelGeneric.mustache